nix/container.nix

```diff
@@ -0,0 +1,66 @@
+{ pkgs, config, ... }:
```

Comment:
Would be nice to update this file with eg. my current configuration.
~~At least we will have somewhere to copy-paste from~~
I will send you the config I worked with.